content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/portenta-x8-fundamentals.md

```diff
@@ -1,85 +1,233 @@
+Benefits of containers include:
 
-### Benefits of Foundries.io
+- **Simplified Development, Testing, and Deployment**: Applications are decoupled from environments, making deployment consistent.
+- **Version Control**: Containers support easy rollbacks if updates fail.
+- **Isolation and Security**: Applications run in isolated environments, enhancing security.
+- **Granularity**: Containers can isolate specific components, improving performance and control.
 
-Foundries.io™ created their custom distribution based on Yocto with minimal software installed, by default implementing top-level cybersecurity features like OP-TEE and OSTREE that makes their solution ideal for professional applications.
+Portenta X8 uses Docker, which employs **Linux Containers (LXC)** for lightweight virtualization and provides a user-friendly interface for container management, with tools like **Dockerfile** for building images, **Docker Daemon** for running them, and orchestration tools like **Docker Swarm** and **Kubernetes** for managing their deployment. Docker containers make applications portable and enhance security by isolating runtime environments.
 
-A custom Over-The-Air (OTA) system update mechanism that is based on a client running on target and a robust Cloud server. And they married Docker-compose as a way to deploy a software solution to a target. This is like having an app store for a particular device with the difference that we're not installing an app but a container that may contain a whole distribution or a minimal distribution running only our app or our set of apps.
+![Docker host](assets/Docker_host.png "Docker host vs client")
 
-Additionally, they developed the Cloud side as well. You can use what's called FoundriesFactory, a Cloud DevSecOps subscription service to build, test, deploy, and maintain secure, updatable IoT and Edge products. It provides a unique id and automatic builds of the base system and containers for this system in one place. Let's now take a look at the Foundries.io Factory page.
+Docker containers ensure portability by packaging applications with their runtime environments, making them easily transferable between machines or cloud environments. The isolation offered by Docker enhances security, preventing threats from propagating across containers or affecting the host system.
 
-### Foundries.io Factory
+![Images and containers](assets/images_containers.png "Images vs containers")
 
-With the help of the Arduino Cloud integration with *Foundries.io*, you can easily create your Factory right from the Arduino Cloud page. You can set your Factory's platform and name. The Portenta X8 will be the platform in this case.
+### Foundries.io Factory and Benefits

```

Comment:
Let's mention somewhere that this is optional and not compulsory for the product to work

Reply:
updated!